e2e/cypress/integration/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const iamVersion = (<string><Object>Cypress.env('IAM_VERSION') ?
+    <string><Object>Cypress.env('IAM_VERSION') :
+    'v2.0');
```

Comment:
[nit] isn't this ternary the same as `<string><Object>Cypress.env('IAM_VERSION') || 'v2.0'`?

Reply:
Ooo if so I was not aware you could use ternaries like that. Will try!